src/go/wsl-helper/cmd/kubeconfig.go

```diff
@@ -57,65 +61,101 @@ var kubeconfigCmd = &cobra.Command{
 		cmd.SilenceUsage = true
 
 		configDir := path.Join(homedir.HomeDir(), ".kube")
-		linkPath := path.Join(configDir, "config")
-		if show {
-			// The output is "true", "false", or an error message for UI.
-			// We will only return nil in this path.
-			target, err := os.Readlink(linkPath)
-			if err != nil {
-				if errors.Is(err, os.ErrNotExist) {
-					fmt.Println("false")
-				} else if errors.Is(err, syscall.EINVAL) {
-					fmt.Printf("File %s exists and is not a symlink\n", linkPath)
-				} else {
-					fmt.Printf("%s\n", err)
-				}
-			} else if target == configPath {
-				fmt.Println("true")
-			} else {
-				// For a symlink pointing elsewhere, we assume we can overwrite.
-				fmt.Println("false")
-			}
-			return nil
+
+		configFile, err := os.Open(configPath)
```

Comment:
With this we don't need the `os.Stat` on line 57 (because if it fails it'll fail here instead).